pom.xml

```diff
@@ -229,6 +229,10 @@
 			<artifactId>scenery</artifactId>
 			<version>0.1.1-SNAPSHOT</version>
 		</dependency>
+		<dependency>
+			<groupId>net.imagej</groupId>
+			<artifactId>imagej-legacy</artifactId>
```

Comment:
Good point! I was trying to stave off the infection, but that file dialog issue is a kicker. However, I do need the file dialog working in the runtime for the standalone projects that use SciView + IJ interactions (it is really tough to remember to add legacy to every single one of those projects), so optional it is.